core/model/src/main/resources/bindings.xjb

```diff
@@ -121,6 +121,7 @@
                  abstract public boolean isEmpty();
 
                  abstract public java.util.List<License> getLicenses();
+                 abstract public java.util.Set<License> getUniqueLicenses();
```

Comment:
My question here is, do we need the old getLicenses method at all. It does not provide any additional value except some redundant listings of licenses. I would rather prefer to change the getLIcenses method to have every license only once to having two methods here. Is there any usage which requires the List as return value or could we change that to Collection?

Reply:
I also believe that the return type of `getLicenses()` should be `Set<License>` but this can cause a lot of changes in the code. Probably we can create a new issue with this topic because it's not in the scope of this PR.

Reply:
That is the point, why I typically prefer Collection as return value, not List and also not Set, because with Collection you have the freedom of returning either a List or a Set. And the real advantage of a List that this is an ordered data structure which you can address by index is nowadays rarely used. In this case, there is no defined order on the licenses returned, so List has no real advantage.

But you are right, we should not change the return value type as of now. So my proposal is, we change the implementation to return in the List every license only once and get rid of getUniqueLicenses. There is no meaning in returning the list of licenses with duplicates. That you use a Set to make each entry unique is actually a implementation detail which should not be visible at the outside. For the implementation that means that you have to convert the Set back to a List. For the amount of licenses involved in an artifact, this should not be too much overhead.